spec/features/attendances/zoom_aliases_correction_spec.rb

```diff
@@ -194,10 +193,10 @@
     student = create(:setup_student) # Need to create one setup student so the module will be considered "set up"
     other_module = student.turing_module
     other_zoom_id = "12345"
-
-    stub_request(:post, ENV['POPULI_API_URL']).
-      with(body: {"task"=>"getCourseInstanceMeetings", "instanceID"=>other_module.populi_course_id}).
-      to_return(status: 200, body: File.read('spec/fixtures/populi/course_meetings.xml'))
+    stub_course_meetings_nil
+    # stub_request(:post, ENV['POPULI_API_URL']).
```

Comment:
can remove commented lines if not needed

Reply:
This is fixed. Thank you!